composer.json

```diff
@@ -31,20 +31,20 @@
         "php-http/httplug-bundle": "^1.10",
         "sensio/framework-extra-bundle": "^4.0",
         "simplethings/entity-audit-bundle": "^1.0",
-        "symfony/console": "4.3.*",
-        "symfony/dotenv": "4.3.*",
-        "symfony/flex": "^1.4",
-        "symfony/form": "4.3.*",
-        "symfony/framework-bundle": "4.3.*",
-        "symfony/ldap": "4.3.*",
+        "symfony/console": "^4.3",
```

Comment:
We should probably change all the libraries to 4.4 as well

Reply:
The up arrow means 4.3 or higher, before we had it set to exactly this version. So we actually have version 4.4.4 installed if you look at the lock file.

Reply:
Yeah, I know the caret means till <5.0.0, thought we can be specific

Reply:
Yes, it wouldn't hurt. I only changed those because otherwise it had upgrade conflicts. Even if there is a 4.5 (if) then it would upgrade to it.  I'm ok with whatever looks best?

Reply:
Actually I take that back, it wouldn't go to 4.5, because in the json file under extra it is constrained to 4.4.